src/webhook/handlers/processGroupEvent.js

```diff
@@ -25,22 +25,17 @@ export default async function processGroupEvent({
   if (type === 'join') {
     // https://developers.line.biz/en/reference/messaging-api/#get-members-group-count
     // Note: leave group cannot get the number
-    const { count: numberOfGroupMembers } = await lineClient.get(
-      `/group/${groupId}/members/count`
-    );
-    console.log(
-      'Joined group, numberOfGroupMembers: ' +
-        JSON.stringify(numberOfGroupMembers)
+    const { count: groupMembersCount } = await lineClient.get(
+      `/${otherFields.source.type}/${groupId}/members/count`
     );
 
     const visitor = ga(groupId, 'N/A', '', otherFields.source.type);
-
+    visitor.set('cm1', groupMembersCount);
```

Comment:
Because chatbot may join/leave a group multiple times,
we should find the last `Join` action of the `Client Id` to get a more accurate `Group Members Count`.
Don't know if there's a better way to handle this condition.

Reply:
Got it, thanks for the note!